k8s/aat/common/cnp/plum-recipe-backend.yaml

```
@@ -28,6 +28,19 @@ spec:
           - recipe-backend-POSTGRES-HOST
           - recipe-backend-POSTGRES-USER
           - recipe-backend-POSTGRES-PASS
+      tests:
+        smoke:
+          once:
+            enabled: true
+            environment:
+              TEST_URL: plum-recipe-backend-java.cnp.svc.cluster.local
```

I would do this:
```suggestion
              TEST_URL: http://plum-recipe-backend-java
```

Not sure if you'll have issues leaving the protocol out, and you shouldn't need the full hostname

---

Good idea, the tests run in the same namespace anyway